actions/enter-date-of-death.vbs

```diff
@@ -0,0 +1,423 @@
+  Loop until err_msg = ""
+  CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+Loop until are_we_passworded_out = false					'loops until user passwords back in
+
+'Convert date of death to MAXIS friendly date
+date_of_death_maxis_format = replace(date_of_death, "/", " ")
+
+'Determine which programs are active or pending
+Call determine_program_and_case_status_from_CASE_CURR(case_active, case_pending, case_rein, family_cash_case, mfip_case, dwp_case, adult_cash_case, ga_case, msa_case, grh_case, snap_case, ma_case, msp_case, emer_case, unknown_cash_pending, unknown_hc_pending, ga_status, msa_status, mfip_status, dwp_status, grh_status, snap_status, ma_status, msp_status, msp_type, emer_status, emer_type, case_status, list_active_programs, list_pending_programs)
+
+'Determine if case is active, if it is active then evaluate what programs are active or pending
+If case_active = True or case_pending = True Then
+  active_pending_case = True
+  'Determine if any CASH programs are active or pending
+  If instr(list_active_programs, "SNAP") OR _
+    instr(list_active_programs, "MSA") OR _
+    instr(list_active_programs, "MFIP") OR _
+    instr(list_active_programs, "GA") OR _
+    instr(list_active_programs, "DWP") OR _
+    instr(list_active_programs, "EGA") OR _
+    instr(list_active_programs, "EA") OR _
+    instr(list_pending_programs, "GA") OR _
+    instr(list_pending_programs, "MSA") OR _
+    instr(list_pending_programs, "MFIP") OR _
+    instr(list_pending_programs, "DWP") OR _
+    instr(list_pending_programs, "FS") OR _
+    instr(list_pending_programs, "MA") OR _
+    instr(list_pending_programs, "MSP") OR _
+    instr(list_pending_programs, "CASH") OR _
+    instr(list_pending_programs, "SNAP") OR _
+    instr(list_pending_programs, "EA") Then
+      active_pending_CASH_SNAP = True
+  End If
+  
+  'Determine if HC is active or pending
+  If instr(list_active_programs, "HC") OR instr(list_pending_programs, "HC") Then
+    active_pending_HC = True
+  End If
+Else
+  active_pending_case = False
+End If
+
+'If case is not active or pending and there are no SNAP, CASH, or HC programs, then script will end
+If active_pending_case = False Then script_end_procedure("The case is not active and no programs are pending. The script will now end.")
+
+'Determine if date of death is prior to arrival date for HH Membs other than 01
+If left(household_member_that_died, 2) <> "01" Then
+  Call navigate_to_MAXIS_screen("STAT", "MEMB")
+  'Navigate to HH Memb that died
+  EMWriteScreen left(household_member_that_died, 2), 20, 76
+  transmit
+  EMReadScreen arrival_date_memb_panel, 8, 4, 73
+  'Convert arrival date to MM/DD/YYYY format
+  arrival_date_memb_panel = Left(replace(arrival_date_memb_panel, " ", "/"), 6) & "20" & right(arrival_date_memb_panel, 2)
```

Comment:
If there is no arrival date you receive an error. Not sure if you'd run into that in real life cases...but might need handling.
![image](https://github.com/user-attachments/assets/97f4de1e-5df7-423d-a0e1-b65ac482a532)

Reply:
Good catch, I resolved this issue.